src/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    margin-right: 4px;
+    height: 16px;
+    width: 16px;
+    background-image: url(../images/star.svg);
```

Comment:
Ensure that the image '../images/star.svg' exists at the specified path. If it doesn't, the background image won't display correctly.